### DISCUSSION TITLE
How to get current pull request number (issue_number)

### DISCUSSION THREAD

**Discussion by virginiarcruz** [octokit/octokit.js#1791] [Q&A] [answered]
I have a repo and I already can create and add labels on GitHub while the Jenkins pipeline is working.

But, in my params, I have issue_number (for example _**856**_) hardcoded, I need to get the dynamic number the current pull request in my pipeline.
Someone can help with this?

```js
const options = {
  owner: args.owner,
  repo: args.repo,
  issue_number: 856,
  labels: ['labelname']
};

octokit.issues.addLabels(options).then(response => console.log('label added: ', response.data));

```

**Comment by gr2m** [comment]
I'm afraid I'm not using Jenkins myself, but usually these builds are triggered by a `pull_request` event, and the payload of that webhook event should be accessible. The payload is documented here:
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#webhook-payload-example-29

**Comment by virginiarcruz** [comment]
So, I have a way that put this label in all the issue_numbers?

**Reply by gr2m** [reply]
If you have access to the payload, you can get the pull request number via

```js
const prNumber = payload.pull_request.number
```

**Comment by gr2m** [comment, marked as answer]
Would this be of any help? https://github.com/jenkinsci/pipeline-github-plugin#pullrequest

**Reply by virginiarcruz** [reply]
I will try with the CHANGE_ID, thanks 😉
